instances.go

```diff
@@ -92,6 +93,17 @@ type InstanceBackup struct {
 	} `json:"schedule,omitempty"`
 }
 
+type instanceDiskEncryption string
+
+func (i *instanceDiskEncryption) String() string {
+	return string(*i)
+}
```

Comment:
Generally we don't implement `.String(...)` methods on our consts but I actually really like this pattern. It might make sense to implement it in some sort of base struct in the future :eyes: